tests/rptest/redpanda_cloud_tests/high_throughput_test.py

```diff
@@ -216,7 +223,9 @@ class HighThroughputTest(PreallocNodesTest):
     # Default value
     msg_timeout = 120
 
-    def __init__(self, test_ctx: TestContext, *args, **kwargs):
+    redpanda: RedpandaServiceCloud
```

Comment:
basically the `.redpanda` member can be either `RedpandaService` or `RedpandaServiceCloud` from the typer checker's point of view (and it's not wrong), but with our giant human brains we know that currently this test only runs in the cloud. So we type hint it at the class level so that we get good type checking in this class.

The need to do this will be obviated by some upcoming changes to the base class structure of the tests.

Reply:
> but with our giant human brains

🤣